packages/calling-stateful-client/review/beta/calling-stateful-client.api.md

```diff
@@ -168,6 +169,11 @@ export interface CaptionsInfo {
 // @public
 export const createStatefulCallClient: (args: StatefulCallClientArgs, options?: StatefulCallClientOptions | undefined) => StatefulCallClient;
 
+// Warning: (ae-internal-missing-underscore) The name "createStatefulCallClientInner" should be prefixed with an underscore because the declaration is marked as @internal
```

Comment:
Internal code should be prefixed with an underscore.
Can we please add a short comment describing these internal functions as well for others.